app/services/analytics_update_service.rb

```diff
@@ -0,0 +1,20 @@
+require 'google/apis/sheets_v4'
+require 'google/apis/analytics_v3'
+require 'base64'
+
+class AnalyticsUpdateService
+  def initialize
+    authorization = Google::Auth::ServiceAccountCredentials.make_creds(
+        json_key_io: StringIO.new(Base64.decode64(ENV['GOOGLE_CREDENTIALS_JSON'])),
+        scope: [Google::Apis::SheetsV4::AUTH_SPREADSHEETS, Google::Apis::AnalyticsV3::AUTH_ANALYTICS]
+    )
+
+    @service = Google::Apis::SheetsV4::SheetsService.new
+    @service.authorization = authorization
+  end
+
+  def add_new_service(api_key, department = 'Not available', service = 'Not available', options = { spreadsheet_id: ENV['SPREADSHEET_ID'], range: ENV['SPREADSHEET_RANGE'] })
```

Comment:
do we want to move getting the these `ENV` variables into Rails config?

Reply:
Are they secret?

Reply:
Not particularly. The spreadsheet id will identify the spreadsheet we're using but it's behind access control anyway.

Reply:
Then I agree with Gids 👍